package.json

```diff
@@ -33,6 +33,9 @@
     "url": "https://github.com/styled-components/css-to-react-native/issues"
   },
   "homepage": "https://github.com/styled-components/css-to-react-native#readme",
+  "jest": {
+    "testURL": "http://localhost"
```

Comment:
What's this?

Reply:
Basically, it use a vhost on the user to connect on my server. Just kidding This is for jest. Try to pull your repo and you will see error such as `localStorage cannot get ....`. We got this in our projects long time ago, seems like you haven't tried it for a while. Try to pull your branch, remove `yarn.lock` and run `npm install && npm test`. You will see.

Reply:
By the way, I've removed the `package-lock.json`, I haven't seen the `yarn.lock`, but it should also be removed from the repo. You don't want to use lock file when you maintain a module.